docs/custom-types.md

```diff
@@ -173,6 +174,8 @@ route:
   regex_priority: 0
   strip_path: false
   preserve_host: true
+  protocols:
+  - http
```

Comment:
Could we add one more element to this array -- `https`?

Reply:
Added!